### PR TITLE
Adds a blue security jacket to sec's wardrobe

### PR DESCRIPTION
**Pull request by Malgover**
## About The Pull Request

Nanotrasen's "Swag & Fashion" department is back again with a fresh blue jacket to wear for the people who prefer blue security.
Original sprite from World Server, I only added the NT logo on the back.

cop1
cop1open
cop2
cop3

cop4
cop5

## Why It's Good For The Game

This PR gives something to wear to people who prefer blue sec without having to resprite the entire department. This is a nice compromise, in my opinion. 

## Changelog
:cl:
add: Adds a casual blue jacket to sec's wardrobe.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
The debate will rage on...blue, red, khaki...

**Comment by Malgover** [comment]
FUCKING CONFLICTS

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -151,6 +151,13 @@
 	inhand_icon_state = "officerbluejacket"
 	body_parts_covered = CHEST|ARMS
 
+/obj/item/clothing/suit/toggle/security/casualofficer
```
Comment (Fikou) [inline code/modules/clothing/suits/jobs.dm:154]: there doesnt seem to be a subtype like toggle/security

Comment (Malgover) [reply code/modules/clothing/suits/jobs.dm:154]: there isn't but there should be

**Comment by Malgover** [comment]
> You currently have a negative Fix/Feature pull request delta of -4. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

Stop texting

Go find God

Come back after you found God

**Comment by Malgover** [comment]
swear to god

**Comment by Malgover** [comment]
i dont get it bruh

**Comment by Malgover** [comment]
what the hell is conflicting? i pulled the most recent versions of **icons/mob/clothing/suit.dmi** and **icons/obj/clothing/suits.dmi**, added the new sprites and it's still conflicting. wtf

**Pull request closed without merging**